nexml_nyiso/notebooks/lstm_mvp.ipynb

```diff
@@ -0,0 +1,670 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\ProgramData\\Miniconda3\\lib\\site-packages\\statsmodels\\tools\\_testing.py:19: FutureWarning: pandas.util.testing is deprecated. Use the functions in the public API at pandas.testing instead.\n",
+      "  import pandas.util.testing as tm\n",
+      "Using plaidml.keras.backend backend.\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "os.environ[\"KERAS_BACKEND\"] = \"plaidml.keras.backend\"\n",
+    "\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from datetime import datetime\n",
+    "from bokeh.plotting import figure, output_file, show\n",
+    "from bokeh.models import ColumnDataSource, DatetimeTickFormatter\n",
+    "from bokeh.models.tools import HoverTool\n",
+    "from bokeh.io.export import get_screenshot_as_png\n",
+    "from bokeh.io import output_notebook\n",
+    "import sklearn\n",
+    "from sklearn.preprocessing import MinMaxScaler, StandardScaler\n",
+    "import seaborn as sns; sns.set()\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import tensorflow as tf\n",
```

Comment:
We should update `requirements.txt` to include TensorFlow or comment out / remove the appropriate lines.